banks-server/Cargo.toml

```diff
@@ -15,6 +15,7 @@ crossbeam-channel = "0.5"
 futures = "0.3"
 solana-banks-interface = { path = "../banks-interface", version = "=1.15.0" }
 solana-client = { path = "../client", version = "=1.15.0" }
+solana-program-runtime = { path = "../program-runtime", version = "=1.15.0" }
```

Comment:
Is this still needed now that you're using the new function on the bank?

Reply:
Nope, removed